.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+<!-- Please describe your changes -->
+
+## How Has This Been Tested?
+<!-- Please describe in detail how you tested your changes. -->
+<!-- Include details of your testing environment, tests ran to see how -->
+<!-- your change affects other areas of the code, etc. -->
+
+## Screenshots (jpeg or gifs if applicable):
+
+## Types of changes
```

Comment:
Screenshots are very helpful indeed. I was talking about the "Types of changes" but the labels permissions is a good point.